src/encoding.md

```diff
@@ -344,6 +345,57 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-invalid-csv]: #ex-invalid-csv
+<a name="ex-invalid-csv"></a>
+## Handle invalid CSV data with Serde
+
+[![csv-badge]][csv] [![serde-badge]][serde] [![cat-encoding-badge]][cat-encoding]
+
+CSV files often contain invalid data. For these cases the csv crate
+provides a custom deserializer, [`csv::invalid_option`], which automatically
+converts invalid data to None values.
+
+[`csv::invalid_option`]: https://docs.rs/csv/*/csv/fn.invalid_option.html
+
+```rust
+# #[macro_use]
+#extern crate error_chain;
+extern crate csv;
+extern crate serde;
+#[macro_use]
+extern crate serde_derive;
+
+#[derive(Debug, Deserialize)]
+struct Record {
+    name: String,
+    #[serde(deserialize_with = "csv::invalid_option")]
+    place: Option<String>,
+    id: u64,
+}
+#
+# error_chain! {
+#     foreign_links {
+#          CsvError(csv::Error);
+#     }
+# }
+
+fn run() -> Result<()> {
+    let data = "name,place,id
+                mark,\"\",46
```

Comment:
the value here is actually valid ( string containing "\"\"").
please note that in the example from docs has a i32 type trying to be deserialized both from such string and xyz which are both invalid i32.

another approach would be to actually leave out one of the fields (remove the `,` too).